docs/deployment_patterns/how_to_use_great_expectations_with_google_cloud_platform_and_bigquery.md

```diff
@@ -324,8 +324,8 @@ We will now take the local GE configuration from [Part 1](#part-1-local-configur
 
 There are a number of ways that Great Expectations can be run in Cloud Composer or Airflow.
 
-1. [Running a Checkpoint in Airflow using a `bash operator`](/docs/deployment_patterns/how_to_use_great_expectations_with_airflow#option-1-running-a-checkpoint-with-a-bashoperator)
-2. [Running a Checkpoint in Airflow using a `python operator`](/docs/deployment_patterns/how_to_use_great_expectations_with_airflow#option-2-running-the-checkpoint-script-output-with-a-pythonoperator)
+1. [Running a Checkpoint in Airflow using a `bash operator`](../deployment_patterns/how_to_use_great_expectations_with_airflow#option-1-running-a-checkpoint-with-a-bashoperator.md)
+2. [Running a Checkpoint in Airflow using a `python operator`](../deployment_patterns/how_to_use_great_expectations_with_airflow#option-2-running-the-checkpoint-script-output-with-a-pythonoperator.md)
```

Comment:
`./how_to_use_great_expectations_with_airflow#option-1-running-a-checkpoint-with-a-bashoperator.md`

`./how_to_use_great_expectations_with_airflow#option-2-running-the-checkpoint-script-output-with-a-pythonoperator.md`

Reply:
I made the fix in new commit.